openmc/summary.py

```diff
@@ -2,17 +2,22 @@
 
 import openmc
 
-try:
-    import h5py
-except ImportError:
-    msg = 'Unable to import h5py which is needed by openmc.summary'
-    raise ImportError(msg)
+
+_h5_imported = False
```

Comment:
I agree with @wbinventor. On subsequent re-imports, if `h5py` is already in the namespace it is essentially a no-op. Also, the error message here doesn't really add much additional information beyond the normal 'No module named h5py' ImportError along with the traceback showing it was called from openmc.summary.